### PR TITLE
Add my first calculator

### PR DESCRIPTION
**Pull request by dcarpenter31**
Added my first calculator, just needs to be approved and merged in!﻿

### CODE REVIEW

**Review by dyc3 — APPROVED** [review]

File: index.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const multiplier = (a: number, b : number) : number => {
+    return a * b;
+}
+
+const divider = (number1 : number, number2 : number) : number => {
+    return number1 / number2
+}
+
+const myCalculator = (a : number, b : number, operation : string) : number => {
```
Comment (dyc3) [inline index.ts:33]: it would be good if `operation` was more strictly typed. something like `"add" | "subtract" | "multiply" | "divider"`

**Review by dcarpenter31 — COMMENTED** [review]
Hey, I got this output, is this right?

Line 9:  Hell oworl!
Line 18:  Does 2 + 2 = 4?
false
Line 52:  undefined

File: index.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+// My first PROGRAM!!!!1
+
+console.log("Line 9: ", "Hell oworl!")
+
+/**
+Adds two numbers together in a really cool way!!! 
+*/
+const superCoolUltraAwesomeAdder = (a : number, b : number) : number => {
+    return a - b;
```
Comment (dcarpenter31) [inline index.ts:15]: hey, should this be a minus? or a plus?

File: index.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+*/
+
+// My first PROGRAM!!!!1
+
+console.log("Line 9: ", "Hell oworl!")
+
+/**
+Adds two numbers together in a really cool way!!! 
+*/
+const superCoolUltraAwesomeAdder = (a : number, b : number) : number => {
```
Comment (dyc3) [inline index.ts:14]: nit: I'd prefer if this name was shorter

File: index.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+const divider = (number1 : number, number2 : number) : number => {
+    return number1 / number2
+}
+
+const myCalculator = (a : number, b : number, operation : string) : number => {
+    let result : number
+    switch (operation) {
+        case "add":
+            result = multiplier(a, b)
```
Comment (dcarpenter31) [inline index.ts:37]:
```suggestion
            result = superCoolUltraAwesomeAdder(a, b)
```

**Review by dyc3 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**